src/extension-runners/firefox-android.js

```diff
@@ -290,6 +291,12 @@ export class FirefoxAndroidExtensionRunner {
         'Firefox for Android target does not support --start-url option.'
       );
     }
+
+    if (this.params.binaryArgs) {
```

Comment:
@rpl I was thinking about smth like

```
const ignoredParams = {
  profilePath: '--profile-path',
  keepProfileChanges: '--keep-profile-changes',
  browserConsole: '--browser-console',
  preInstall: '--pre-install',
  startUrl: '--start-url',
  binaryArgs: '--binary-args',
};

const getIgnoredParamsWarningsMessage = (optionName) => {
  return `Firefox for Android target does not support ${optionName} option`;
};
```

so function `printIgnoredParamsWarnings` can be a bit cleaner

```
printIgnoredParamsWarnings() {
  Object.entries(ignoredParams).forEach((ignoredParam) => {
      if (this.params[ignoredParam]) {
        log.warn(
          getIgnoredParamsWarningsMessage(ignoredParams[ignoredParam])
        );
      }
    });
}
```

Only thing I stuck is `flow` with some errors, but I can dig in if you approve this approach.

Reply:
:+1: I like the idea

Reply:
(let me know if you get stuck on issues with flow errors, I'll be happy to help you to figure out how to properly handle those)